dunstctl

```diff
@@ -83,6 +84,17 @@ case "${1:-}" in
 			property_set paused variant:boolean:"$2"
 		fi
 		;;
+	"status")
```

Comment:
`status` is a bit ambiguous, one could assume this is paused status. I'd suggest `count` as the command (unless you have a better idea?)

Reply:
I have no better idea as `count`. I'll change it…